src/main/java/com/mathworks/ci/BuildArtifactAction.java

```diff
@@ -220,6 +220,28 @@ private void iterateAllTaskAttributes(Entry pair, BuildArtifactData data) {
             case "skipped":
                 data.setTaskSkipped((Boolean) pair.getValue());
                 break;
+            case "skipReason":
+                String skipReasonKey = pair.getValue().toString();
+                String skipReason;
+                switch(skipReasonKey){
+                    case "UpToDate":
```

Comment:
After consulting the MATLAB editor, I think it's best not to capitalize the text in parentheses. 

Here is a list of reasons that the build tool displays:

* Skipped mytask (user requested)
* Skipped mytask (up-to-date)
* Skipped mytask (dependency failed)
 
To be consistent, the text should be (user requested).

Reply:
Updated texts to lowercase. Initially I thought I'll update the text colour with my test results view PR but now I'm planning to check with Mayank on it for both Jenkins, GitHub (I have a PR open at the moment for GitHub as well so might as well add it their if any change is required)